tdrs-backend/vault/vault.hcl

```diff
@@ -0,0 +1,19 @@
+# Enable Vault's web UI
```

Comment:
vault secrets can be versioned as well, question is do want to use versioned secrets? Basically any changes to secrets will create a new version, but we still have access to previous versions.